lib/ansible/modules/crypto/luks_device.py

```diff
@@ -64,6 +64,12 @@
             - "BEWARE that working with keyfiles in plaintext is dangerous.
               Make sure that they are protected."
         type: path
+    keysize:
+        description:
+            - "Sets the key size only if LUKS container does not exist."
+        type: int
+        default: 512
```

Comment:
How about not having a default in the module, and letting `cryptsetup` use its internal default if the user doesn't specify something?

Reply:
I think it's a good idea indeed!